test/e2e/run_test.sh

```diff
@@ -28,7 +28,25 @@ function setup_kind_cluster() {
     fi
 
     # Create KinD cluster.
-    cat <<EOF | kind create cluster --name="${NAME}" -v4 --retain --image "${IMAGE}" --config=-
+
```

Comment:
You should also add a line here for ipv6 e2e call:

https://github.com/kmesh-net/kmesh/blob/main/Makefile#L149